pandas/io/tests/parse_dates_tests.py

```diff
@@ -0,0 +1,471 @@
+"""
+Tests date parsing functionality for all of the
```

Comment:
well you need to fix it then if that's the way you want it. the correct way to do this is to split off the base class into ``common.py`` or something which is then imported into each file. Look at how ``tests/indexes`` was done